src/poojaJoshi/Test1_NonRepetingString.java

```diff
@@ -0,0 +1,31 @@
+package poojaJoshi;
+
+public class Test1_NonRepetingString {
+	
+		int getcharfreq(String name ,char ch) {
+			int count = 0;
+			for(int index = 0;index<name.length();index++) {
+				if(name.charAt(index) == ch)
+					count++;		
+			}
+			return count;
+		}
+
+		void findNonRepChar(String name) {
+			char chr =' ';
+			for(int index=0;index<name.length();index++) {
+				char ch =name.charAt(index);
+				int count =getcharfreq(name ,ch);
+				if(count == 1) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop